kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/MessagePublishContext.java

```diff
@@ -68,6 +69,19 @@ public static MessagePublishContext get(CompletableFuture<Long> offsetFuture,
         return callback;
     }
```

Comment:
I think the old `get` method could be removed, right?

Reply:
will remove the legacy code